### PR TITLE
fix: Ammend get instance role

### PR DESCRIPTION
**Pull request by fmubaidien**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -167,7 +167,7 @@ func updateRoleMappings() {
 	roleMappingsUpdated := transformRoleMappings(roleMappings, awsIAMRoles, accountId)
 
 	// Add worker node role bindings if those are absent
-	iamRoleARN := getInstanceRole()
+	iamRoleARN := "arn:aws:iam::" + accountId + ":role/"getInstanceRole()
```
Comment (justinas-b) [inline main.go:170]:
```suggestion
	iamRoleARN := "arn:aws:iam::" + accountId + ":role/" + getInstanceRole()
```

Comment (justinas-b) [reply main.go:170]: I think using plus signs will add spaces between each component, would it make more sense to use `fmt.Splrintf()`?

File: kubernetes_test.go [hunk 2]
```diff
@@ -373,4 +373,46 @@ func TestTransformRoleMappings(t *testing.T) {
...
+			},
+			{
+				RoleARN:       "arn:aws:iam::123456789012:role/node-role",
+				PermissionSet: "",
+				Username:      "",
+				Groups:        []string{},
+			},
+		}
+		iamRoleARN := "arn:aws:iam::" + "123456789012" + ":role/" + getInstanceRole()
+		mappings = addWorkerNodeRoleBindings(roleMappingsUpdated, iamRoleARN)
```
Comment (justinas-b) [inline kubernetes_test.go:411]:
```suggestion
		mappings = addWorkerNodeRoleBindings(mappings, iamRoleARN)
```

File: kubernetes_test.go [hunk 3]
```diff
@@ -373,4 +373,46 @@ func TestTransformRoleMappings(t *testing.T) {
...
+			},
+			{
+				RoleARN:       "arn:aws:iam::123456789012:role/node-role",
+				PermissionSet: "",
+				Username:      "",
+				Groups:        []string{},
+			},
+		}
+		iamRoleARN := "arn:aws:iam::" + "123456789012" + ":role/" + getInstanceRole()
+		mappings = addWorkerNodeRoleBindings(mappings, iamRoleARN)
```
Comment (justinas-b) [inline kubernetes_test.go:411]: to test `getInstanceRole()`, we would need to mock aws-sdk lib, though to simplify it, maybe we could just do:
```suggestion
		iamRoleARN := "arn:aws:iam::123456789012:role/node-role"
		mappings = addWorkerNodeRoleBindings(mappings, iamRoleARN)
```

Comment (fmubaidien) [reply kubernetes_test.go:412]: How about that instead?

**Pull request merged**